cardano-db/src/Cardano/Db/Schema.hs

```diff
@@ -136,5 +143,108 @@ share
     poolHash            ByteString          sqltype=hash32type
     UniquePoolOnData    poolHash
 
+  ------------------------------------------------------------
+  -- Shelley bits
+  ------------------------------------------------------------
+
+  CentralFunds
+    epochNo             Word64
+    treasury            Word64              sqltype=lovelace
+    reserves            Word64              sqltype=lovelace
+    UniqueCentralFunds  epochNo
+
+  Reward
+    rewardAddress       ByteString          sqltype=staking_addr_hash
+    instantaneous       Bool
+    poolId              ByteString          sqltype=hash32type
+    epochNo             Word64
+    reward              Word64              sqltype=lovelace
+    UniqueReward        rewardAddress poolId epochNo instantaneous
+
+  Delegation
+    rewardsAddress      ByteString          sqltype=staking_addr_hash
+    poolId              ByteString          sqltype=hash32type
+    slotNo              Word64
+    amount              Word64              sqltype=lovelace
+    inTx                TxId
+    UniqueDelegation    rewardsAddress poolId inTx
+
+  StakeKeyRegistration
+    stakekeyAddress     ByteString          sqltype=staking_addr_hash
+    slotNo              Word64
+    inTx                TxId
+    UniqueStakeKeyRegistration stakekeyAddress inTx
+
+  StakeKeyDeregistration
+    stakekeyAddress     ByteString          sqltype=staking_addr_hash
+    slotNo              Word64
+    inTx                TxId
+    UniqueStakeKeyDeregistration stakekeyAddress inTx
+
+  ScriptRegistration
+    scriptAddress       ByteString          sqltype=script_hash
+    slotNo              Word64
+    inTx                TxId
+    UniqueScriptRegistration scriptAddress inTx
+
+  ScriptDeregistration
+    scriptAddress       ByteString          sqltype=script_hash
+    slotNo              Word64
+    inTx                TxId
+    UniqueScriptDeregistration scriptAddress inTx
+
+  ParameterUpdate
+    epochNo             Word64
+    minFee              Word64
+    maxFee              Word64
+    maxBlockSize        Word64
+    maxTxSize           Word64
+    maxBhSize           Word64
+    keyDeposit          Word64              sqltype=lovelace
+    keyDepositRefund    Word64              sqltype=interval
+    keyDepositDecay     Word64              sqltype=rational
+    poolDeposit         Word64              sqltype=lovelace
+    poolDepositRefund   Word64              sqltype=interval
+    poolDepositDecay    Word64              sqltype=rational
+    maxEpoch            Word64
+    nOptimal            Word64
+    influence           Word64              sqltype=rational
+    monetaryExpandRate  Word64              sqltype=interval
+    treasuryGrowthRate  Word64              sqltype=interval
+    activeSlotCoeff     Word64              sqltype=interval
+    decentralisation    Word64              sqltype=interval
```

Comment:
I thought that was a floating point value in the range `[0 .. 1]`

Reply:
@kevinhammond The interval type just defines the minimum and the maximum. Do we need this to be a floating-point or a 0 - 100 integer?